WcaOnRails/app/models/competition.rb

```diff
@@ -29,6 +29,8 @@ class Competition < ActiveRecord::Base
            allow_nil: true,
            with_model_currency: :currency_code
 
+  scope :present, -> { where("CAST(CONCAT(endYear,'-',endMonth,'-',endDay) as Datetime) >= ?", Date.today) }
```

Comment:
`present` seems like the wrong name, since this would return all future competitions as well.

Coming up with a better name is harder. `not_ended` is correct, but awkward.